vtStorAta/DriveAtaCommandExtensions.cpp

```diff
@@ -50,7 +50,7 @@ eErrorCode IssueCommand_IdentifyDevice( std::shared_ptr<cDriveInterface> Drive,
 }
 
 
-eErrorCode IssueCommand_ReadDma(std::shared_ptr<cDriveInterface> Drive, U32 CommandType, std::shared_ptr<cBufferInterface> Data, U32 Lba, U8 Count)
+eErrorCode IssueCommand_ReadDma(std::shared_ptr<cDriveInterface> Drive, U32 CommandType, std::shared_ptr<cBufferInterface> Data, U32 Lba, U32 Count)
```

Comment:
I think U16 is enough for current requirements, but U32 is safer in the case we want to extend.